provider/cmd/pulumi-gen-eks/main.go

```diff
@@ -1559,6 +1563,19 @@ func genDotNet(pkg *schema.Package, outdir string) {
 	mustWriteFiles(outdir, files)
 }
 
+func genGo(pkg *schema.Package, outdir string) {
+	files, err := gogen.GeneratePackage(Tool, pkg)
+	if err != nil {
+		panic(err)
+	}
+
+	// The Go code generator emits a provider file that we don't need, so remove it.
+	// This should be an option passed to the code generator, but we'll make the tweak here for now.
+	// delete(files, "eks/provider.go")
```

Comment:
Yeah, let's just delete these lines. Originally I was thinking we didn't need the provider, and had stripped it from Python and .NET, but I'm going to change Python and .NET so that we keep it.